scripts/bintray.sh

```diff
@@ -3,8 +3,21 @@ set -e
 
 OUTPUT=${1:-out}
 
+if curl -s -I "https://api.bintray.com/repos/$BINTRAY_PROJECT" | grep -q 404; then
+    echo "package repository not found, creating repository"
+    curl -u$BINTRAY_USER:$BINTRAY_API_KEY -X POST -I "https://api.bintray.com/repos/$BINTRAY_PROJECT"
+fi
+
 for PKG in $(ls $OUTPUT); do
+    if curl -s -I "https://api.bintray.com/packages/$BINTRAY_PROJECT/$PKG" | grep -q 404; then
+        echo "package $PKG not found, creating package "
+        PKG_NAME=$(echo $PKG | sed -E "s|^(([a-zA-Z\-]+)-([0-9\.\-]+)\..*)$|\2|")
+        PKG_METADATA='{"name": "$PKG_NAME", "licenses": ["Apache-2.0"], "vcs_url": "https://github.com/mantl-pkg/stub-url"}'
```

Comment:
`github.com/mantl-pkg/stub-url`? Can we grab the actual directory from somewhere, or just point it to the root of this repo?